goblin/driver/cluster.py

```diff
@@ -165,8 +166,15 @@ def _get_message_serializer(self, config):
             config['message_serializer'] = my_import(message_serializer)
         return config
 
-    def config_from_module(self, filename):
-        raise NotImplementedError
+    def config_from_module(self, module):
+        if isinstance(module, str):
+            module = importlib.import_module(module)
+        config = dict()
+        for item in dir(module):
+            if not item.startswith('_') and item.lower() in self.DEFAULT_CONFIG:
```

Comment:
It just short-circuits the expression, so it won't transform the string and check for membership in DEFAULT_CONFIG at all if it starts with '_'. Admittedly it is a bit nitpicky, the performance savings is likely inconsequential.